tools/bitfields.lisp

```diff
@@ -72,6 +72,22 @@
    "MAP_INVALIDATE_RANGE_BIT"
    "MAP_WRITE_BIT"
    "MAP_READ_BIT"))
+
+
+ (:enum-type

```

Comment:
This doesn't seem to affect generated files. When I run stuff from tools only timestamps are changed in files. Need advice.